tests/flow/test_maxflow.py

```
@@ -0,0 +1,1346 @@
+                   defaultCapacity:-1})
+               YIELD maxFlow""",
+        ]
+
+        for q in invalid_queries:
+            try:
+                self.graph.query(q)
+                self.env.assertFalse(True)
+            except Exception:
+                # Each query exercises a different config-validation path; a
+                # server-side error is the only acceptable outcome here.
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)